index.html

```diff
@@ -2128,6 +2260,209 @@ <h4>Bidirectional text</h4>
+        <p>A processor that takes as its input a <a>DAPT document</a> that
+          contains vocabulary relating to features that it does support,
+          but where support for those features is excluded from the content profiles
+          to which the document claims conformance, SHOULD NOT implement
+          those features in the context of that document.</p>
+      </section>
+      <section>
+        <h3>Handling <code>&lt;div&gt;</code> and  <code>&lt;p&gt;</code> elements</h3>
+        <p>[[TTML2]] allows <code>&lt;div&gt;</code> elements to contain any combination of
+          <code>&lt;div&gt;</code> elements and <code>&lt;p&gt;</code> elements.
+          The DAPT data model only describes the case where the <code>&lt;body&gt;</code> element
+          contains a <code>&lt;div&gt;</code> element that contains zero or more <code>&lt;p&gt;</code> elements.
+          In addition, attributes not corresponding to properties in the DAPT data model are permitted.</p>
+        <p>This gives rise to possibilities such as:</p>
+        <ul>
+          <li><code>&lt;div&gt;</code> elements that contain children that are not <a>Text</a> objects,
+            for example other <code>&lt;div&gt;</code> elements or <code>&lt;metadata&gt;</code> elements;</li>
+          <li><code>&lt;div&gt;</code> elements that do not have attributes required for <a>Script Events</a>, like <code>xml:id</code>;</li>
+          <li><code>&lt;div&gt;</code> elements that have attributes absent from the DAPT data model for <a>Script Events</a>,
+            like <code>xml:space</code>;</li>
+          <li><code>&lt;p&gt;</code> elements whose parent <code>&lt;div&gt;</code> element does not appear to be a <a>Script Event</a>.</li>
+        </ul>
+
+        <p>The following processing rules resolve these cases:</p>
+        <ol>
+          <li>A <code>&lt;div&gt;</code> element that contains any <code>&lt;div&gt;</code> element children
+            MUST NOT be mapped to a <a>Script Event</a>;
+            the processor instead MUST iterate through those <code>&lt;div&gt;</code> element children
+            (recursively) and consider if each one meets the requirements of a <a>Script Event</a>;</li>
+          <li>A <code>&lt;div&gt;</code> element that contains unrecognised attributes
+            MUST NOT be rejected as a <a>Script Event</a> if it otherwise meets the requirements
+            for being a <a>Script Event</a>, such as having a valid <code>xml:id</code>
+            representing the <a>Script Event Identifier</a>;</li>
+          <li>A <code>&lt;p&gt;</code> element that is not a child of a <code>&lt;div&gt;</code> element
+            that maps to a <a>Script Event</a> MUST NOT be mapped to a <a>Text</a> object;</li>
+          <li>A <code>&lt;p&gt;</code> element that is a child of a <code>&lt;div&gt;</code> element
+            that maps to a <a>Script Event</a> MUST be mapped to a <a>Text</a> object.</li>
+        </ol>
```

Comment:
Rules are negative (MUST NOT) and it is not clear how to clearly identify a script event.

Reply:
Fixed in 6c1b4ab.